server/base/src/main/java/org/apache/accumulo/server/util/FileUtil.java

```diff
@@ -163,15 +163,15 @@ private static Path createTmpDir(AccumuloConfiguration acuConf, VolumeManager fs
         try {
           if (reader != null)
             reader.close();
-        } catch (IOException e) {
+        } catch (Exception e) {
```

Comment:
No they are not needed for compilation but since the code was failing quietly before, I didn't want to introduce an exception to classes that use FileUtil.